stdlib/asconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "options": {
+    "-O3z": true,
```

Comment:
Btw it was `-O3` before. It's could significant affected on performance for `memory.copy` / `memory.compare`

Reply:
Ah, good catch.

Reply:
@MaxGraey How so?  Is this a binaryen issue?

Reply:
> How so?

AssemblyScript uses conditionally compiled branches specialized for speed and / or size. Sometimes for good size reduction we use compleately different implementations.

>  Is this a binaryen issue?

Binaryen's also sensitive to level of shrinkage. Especially on inlining